helpers/dispatchRenderEvent.js

```diff
@@ -0,0 +1,9 @@
+export function dispatchRenderEvent(elem) {
+	const e = new CustomEvent('d2l-element-render', {
+		bubbles: false,
+		composed: false
+	});
```

Comment:
@dbatiste I centralized this as I'm sure we'll need to use it elsewhere for any test that needs to wait for the component to render before doing stuff. Simply waiting for `await elem.updateComplete` isn't enough since it resolves whether `shouldUpdate` returns `true` or `false`.

Thoughts about bubbling and composed? I figured folks would want to listen for a very specific element's render event, and wouldn't want sub-elements' events bubbling up and triggering.

Reply:
Is the `requestAnimationFrame` so that the event is dispatched **after** rendering has happened, presuming the user is calling `dispatchRenderEvent` from inside their `render`?

With the current approach, the tests (say for `d2l-more-less`) are coded to know the fact that it will dispatch the event, and in it's case, is probably fine.  However, what if an ancestor wants to know when **all** elements have rendered, and it doesn't know the inner working of the components; i.e. whether they have any async dependency?

Spit-balling again. What if the event is dispatched when a component knows it's going to do async work before it renders, and the event carries a promise that is resolved when the work is complete?  Justin describes this approach in his "Pending State Protocol", and further provides a "PendingStateContainerMixin" that listens for those events and changes views when all promises are resolved.

The event would become something like:
```
export function dispatchPendingRender(elem, pendingPromise) {
   const e = new CustomEvent('d2l-element-render', {
      bubbles: true,
      composed: true,
      detail: { promise: pendingPromise }
   });
   requestAnimationFrame(
      () => elem.dispatchEvent(e)
   );
}
```

Reply:
I like that -- I'll update it.